### PR TITLE
implemented game operations

### PR DESCRIPTION
**Pull request by dten1**

### CODE REVIEW

File: business/src/main/java/com/tinqin/academy/business/operations/game/CreateGameOperationProcessor.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CreateGameOperationProcessor implements CreateGameOperation {
+
+    private final GameRepository gameRepository;
+    private final CategoryRepository categoryRepository;
```
Comment (GeorgiMihaylov23) [inline business/src/main/java/com/tinqin/academy/business/operations/game/CreateGameOperationProcessor.java:21]: categoryRepository isn't used

**Review by gerganovalexander — APPROVED** [review]
@Data annotation could be refactored to separate annotations. GG WP

**Review by GeorgiMihaylov23 — APPROVED** [review]
Brao mitak

**Review by gerganovalexander — APPROVED** [review]
Муцка

**Pull request merged**